SparkPost/matchbox
# FE-925 Update Button Component

Give your PR a recognizable title. For example: **FE-123: Ticket Title** or **Resolve Issue #123: Description**.

### What Changed
- Add new prop outlineBorder
- Change border styles to match border on TextField
- Fix system props
<img width="742" alt="Screen Shot 2020-03-31 at 5 14 44 PM" src="https://user-images.githubusercontent.com/3878251/78080539-11c8e300-7374-11ea-9402-e5bda314be5c.png">
<img width="770" alt="Screen Shot 2020-03-31 at 5 14 27 PM" src="https://user-images.githubusercontent.com/3878251/78080548-1392a680-7374-11ea-84f8-1752710e5adf.png">

### How To Test or Verify
- Run storybook `npm run start:storybook`
- Check stories to make sure styles are as intended
### PR Checklist
